creator-node/src/monitors/ipfs.js

```diff
@@ -0,0 +1,63 @@
+const { ipfs } = require('../ipfsClient')
+
+/**
+ * Performs a diagnostic test on IPFS operations to
+ * confirm functionality:
+ *   - Adds a file
+ *   - Gets a file
+ */
+const getIPFSReadWriteStatus = async () => {
+  try {
+    const start = Date.now()
+    const timestamp = start.toString()
+    const content = Buffer.from(timestamp)
+
+    // Add new buffer created from timestamp (without pin)
+    const results = await ipfs.add(content, { pin: false })
+    const hash = results[0].hash // "Qm...WW"
+
+    // Retrieve and validate hash from local node
+    const ipfsResp = await ipfs.get(hash)
+    const ipfsRespStr = ipfsResp[0].content.toString()
+    if (ipfsRespStr !== timestamp) {
+      throw new Error('Read bytes differ from written bytes')
+    }
+
+    const duration = `${Date.now() - start}ms`
+
+    return JSON.stringify({ hash, duration })
+  } catch (e) {
+    return null
+  }
+}
+
+/**
+ * Performs a diagnostic test on IPFS operations to
+ * confirm functionality:
+ *   - Pins a file
+ */
+const getIPFSPinStatus = async () => {
```

Comment:
we can actually just remove this now since we never pin, but maybe just leave this function in (with a comment) and never actually consume it in monitoring

Reply:
done! thx

Reply:
👍